pkg/vz/vm_darwin.go

```diff
@@ -556,6 +560,28 @@ func attachFolderMounts(driver *driver.BaseDriver, vmConfig *vz.VirtualMachineCo
 	return nil
 }
 
+func attachAudio(driver *driver.BaseDriver, config *vz.VirtualMachineConfiguration) error {
+	if *driver.Yaml.Audio.Device == "vz" {
+		outputStream, err := vz.NewVirtioSoundDeviceHostOutputStreamConfiguration()
+		if err != nil {
+			return err
+		}
+		inputStream, err := vz.NewVirtioSoundDeviceHostInputStreamConfiguration()
```

Comment:
didn't configure any input stream for qemu, but that is probably a good idea... maybe a boolean ? or not

Reply:
I think it should be fine as QEMU directly has a device that support both. But i haven't tested microphone case.

I think it needs some permission (NSMicrophoneUsageDescription) in Info.plist. Checking on how to provide that.

Reply:
Just verified Microphone as well, it is working without any extra Info.plist. Not sure how but working :D

Reply:
Ah this makes a lot of sense.
Thanks for the clarification 😃